.github/bors.toml

```diff
@@ -0,0 +1,4 @@
+required_approvals = 1
+block_labels = ["wip"]
+delete_merged_branches = true
+status = ["ci-linux (stable)", "ci-linux (nightly, true)"]
```

Comment:
do we want to block bors merging on nightly passing?

Reply:
Oh, no I guess we should remove that requirement :)

Reply:
Fixed!